lib/utils/note-utils.js

```diff
@@ -27,17 +27,23 @@ const noteTitleAndPreviewMdRegExp = /[# ]*(\S[^\n]*)\s*(\S[^\n]*)?/;
  */
 export const noteTitleAndPreview = note => {
   const content = note && note.data && note.data.content;
-  const isMarkdown =
-    note && note.data && note.data.systemTags.some(tag => tag === 'markdown');
 
-  const match = isMarkdown
+  const match = isMarkdown(note)
     ? noteTitleAndPreviewMdRegExp.exec(content || '')
     : noteTitleAndPreviewRegExp.exec(content || '');
 
-  const title = (match && match[1] && match[1].slice(0, 200)) || 'New note...';
-  const preview = (match && match[2]) || '';
+  const title = (match && match[2] && match[2].slice(0, 200)) || 'New note...';
+  const preview = (match && match[3]) || '';
 
   return { title, preview };
 };
 
+export const isMarkdown = note => {
+  return (
+    note &&
+    note.data &&
+    note.data.systemTags.includes(tag => tag === 'markdown')
+  );
+};
+
```

Comment:
oh thanks! sorry I didn't mean to imply this had to be done here in this PR, but I guess it's in a reasonable spot for it

Reply:
That was only moving few lines to the separate method. I did while making other changes.